appserver/tests/payara-samples/classpath/embedded-vs-jersey/pom.xml

```diff
@@ -56,40 +56,4 @@
             <artifactId>junit</artifactId>
         </dependency>
     </dependencies>
-
-    <profiles>
-        <profile>
-            <id>payara4</id>
-            <activation>
-                <property>
-                    <name>payara.version</name>
-                    <value>/4.+/</value>
-                </property>
-            </activation>
-            <dependencies>
-                <dependency>
-                    <groupId>fish.payara.arquillian</groupId>
-                    <artifactId>payara-client-ee7</artifactId>
-                    <scope>test</scope>
-                </dependency>
-            </dependencies>
-        </profile>
-    
-        <profile>
-            <id>payara5</id>
-            <activation>
-                <property>
-                    <name>payara.version</name>
-                    <value>/5.+/</value>
-                </property>
-            </activation>
-            <dependencies>
-                <dependency>
-                    <groupId>fish.payara.arquillian</groupId>
-                    <artifactId>payara-client-ee8</artifactId>
-                    <scope>test</scope>
-                </dependency>
-            </dependencies>
```

Comment:
the dependency was part of test, should remain here.

Reply:
It was not used. Currently we have reported issue starting from 5.193.1, test passes on 5.193 and 5.192, so we broke it. Now is not possible to use embedded payara (web/all) with jersey-client and even it's own jersey-client. Fails on some runlevel issue in request tracing filter.
I will try to fix it next week under CUSTCOM-70

Reply:
I'm fairly sure that's a legit issue that this test tests for 😉

Reply:
It is and it is broken, as I wrote, there already exists issue for that, CUSTCOM-70. The dependency was not used, tests work same with or without it (passes for 5.192, 5.193, fails for 5.193.1, 5.194 and master).